common/src/main/java/io/netty/util/internal/InternalThreadLocalMap.java

```diff
@@ -164,13 +164,17 @@ public int size() {
     }
 
     public StringBuilder stringBuilder() {
-        StringBuilder builder = stringBuilder;
-        if (builder == null || builder.capacity() > STRING_BUILDER_MAX_CAPACITY) {
```

Comment:
@jasontedor I wonder if we should allow to adjust the max size via a system property ?